src/utils/package-json.ts

```diff
@@ -0,0 +1,97 @@
+/*!
+ * Copyright (c) 2019 SAP SE or an SAP affiliate company. All rights reserved.
+ */
+import cli from 'cli-ux';
+import * as execa from 'execa';
+import * as fs from 'fs';
+import * as path from 'path';
+
```

Comment:
this is mostly moved code, right?

Reply:
Yes, it's all moved but also refactored to some degree.